scripts/Copyright-Checker/copyright-checker.sh

```diff
@@ -62,20 +62,35 @@ function is_copyright_valid() {
   local file_extension="${file##*.}"
   local offset=1
 
-  # since PSR-12 the php files will contain the copyright license as
-  # document-level comment, which starts on line 3.
-  if [ "php" = "${file_extension}" ]; then
-    offset=3
-  fi
+  is_ui_example ${file}
+  local is_example=${?}
 
-  for copyright_line in "${COPYRIGHT_LINES[@]}"; do
-    local line_to_check="$(sed "${offset}q;d" "${file}")"
-    if ! [ "${copyright_line}" = "${line_to_check}" ]; then
-      return 1
+  if [ 1 -eq ${is_example} ]; then
+    if [ "php" = ${file_extension} ]; then
+      offset=3
     fi
-
-    offset=$((1 + ${offset}))
-  done
+    for copyright_line in "${COPYRIGHT_LINES[@]}"; do
```

Comment:
Please ignore this comment if you think is is out of the scope of this PR, I just want to point this out for completeness:
`$foo[@]` is a bash specific feature I would change the `COPYRIGHT_LINES` to
``` sh
COPYRIGHT_LINES=`cat <<EOF
/**
 * This file is part of ILIAS, a powerful learning management system
...
EOF
```
```suggestion
    echo "$COPYRIGHT_LINES" | while read copyright_line; do
```